cmd/open.js

```diff
@@ -40,7 +31,11 @@ exports.handler = (argv) => {
 };
 
 function getService(argv, res) {
-    if (argv.npms) { return `https://npms.io/search?term=${argv.package}`; }
-    if (argv.npm) { return res.body.collected.metadata.links.npm; }
-    return res.body.collected.metadata.links.repository || res.body.collected.metadata.links.npm;
+    const links = res.body.collected.metadata.links;
+
+    if (argv.link) {
```

Comment:
@satazor What about backwards compatible support?

```
$ npms open gulp
```

Should this no longer work as it does now (since `--link` is not supplied)